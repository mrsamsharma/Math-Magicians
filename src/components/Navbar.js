import React from 'react';
import '../styles.css';
import { Link } from 'react-router-dom';

export default function Navbar() {
  return (
    <nav className="navbar navbar-expand-lg bg-light">
      <div className="container-fluid">
        <a className="navbar-brand" href="/">Math Magicians</a>
        <button className="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
          <span className="navbar-toggler-icon" />
        </button>
        <div className="collapse navbar-collapse flex-end" id="navbarNav">
          <ul className="navbar-nav">
            <li className="nav-item">
              <Link className="nav-link active" aria-current="page" to="/">Home</Link>
            </li>
            <li className="nav-item">
              <Link className="nav-link active" to="/calculator">Calculator</Link>
            </li>
            <li className="nav-item">
              <Link className="nav-link active" to="/quote">Quote</Link>
            </li>
          </ul>
        </div>
      </div>
    </nav>
  );
}
