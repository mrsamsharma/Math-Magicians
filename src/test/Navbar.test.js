import React from 'react';
import Navbar from '../components/Navbar';
import renderer from 'react-test-renderer';
import { MemoryRouter as Router } from 'react-router-dom';

it('Digit Component renders correctly', () => {
    const tree = renderer
      .create(<Router>
        <Navbar />
      </Router>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });