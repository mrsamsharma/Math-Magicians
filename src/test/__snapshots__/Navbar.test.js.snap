// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Digit Component renders correctly 1`] = `
<nav
  className="navbar navbar-expand-lg bg-light"
>
  <div
    className="container-fluid"
  >
    <a
      className="navbar-brand"
      href="/"
    >
      Math Magicians
    </a>
    <button
      aria-controls="navbarNav"
      aria-expanded="false"
      aria-label="Toggle navigation"
      className="navbar-toggler"
      data-bs-target="#navbarNav"
      data-bs-toggle="collapse"
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <div
      className="collapse navbar-collapse flex-end"
      id="navbarNav"
    >
      <ul
        className="navbar-nav"
      >
        <li
          className="nav-item"
        >
          <a
            aria-current="page"
            className="nav-link active"
            href="/"
            onClick={[Function]}
          >
            Home
          </a>
        </li>
        <li
          className="nav-item"
        >
          <a
            className="nav-link active"
            href="/calculator"
            onClick={[Function]}
          >
            Calculator
          </a>
        </li>
        <li
          className="nav-item"
        >
          <a
            className="nav-link active"
            href="/quote"
            onClick={[Function]}
          >
            Quote
          </a>
        </li>
      </ul>
    </div>
  </div>
</nav>
`;
